x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -23484,4 +23484,4 @@
     "xpack.watcher.watchEdit.thresholdWatchExpression.aggType.fieldIsRequiredValidationMessage": "フィールドを選択してください。",
     "xpack.watcher.watcherDescription": "アラートの作成、管理、監視によりデータへの変更を検知します。"
   }
-}
\ No newline at end of file
+}
```

Comment:
these translation files don't have any actual changes?

Reply:
oh, 🤔 I wonder if it's because they haven't been translated yet? Looking at https://raw.githubusercontent.com/elastic/kibana/c6af17faeeea3010d229d9b7a0e77c2a89d6d184/x-pack/plugins/translations/translations/zh-CN.json I don't see the one I changed or a few other recent additions.

I ran `node scripts/i18n_check --fix` before submitting and it succeeded. I'm going to merge this as-is and circle back to figuring out if anything else is missing.